python/cugraph/cugraph/sampling/node2vec.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
should be 2022-2023
I think.

Reply:
thanks - I was in a flow of just updating the date and missed that this should have been a range